pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -314,7 +317,7 @@ func (c *Cluster) handleRemovedNodes(config *provisionConfig) {
 
 	for removedNode, osdDeployments := range removedNodes {
 		logger.Infof("processing removed node %s", removedNode)
-		if err := c.isSafeToRemoveNode(removedNode, osdDeployments); err != nil {
+		if err = c.isSafeToRemoveNode(removedNode, osdDeployments); err != nil {
```

Comment:
This change is unnecessary.

Reply:
My linter is warningly complaining about `err` being redeclared here, I'll remove the change though I think I'll soon finally create a doc where we put in points such as what linters to use/run and what code design points to follow.